Makefile

```diff
@@ -15,4 +25,23 @@ update-charm-libs:
 check-dashboard-updates:
 	./scripts/check-dashboard-updates.sh
 
-.PHONY: help update-charm-libs check-dashboard-updates
+clean:
+	@echo "clean files"
+	@git clean -ffXd -e '!.idea' -e '!.vscode'
```

Comment:
Why we need this? I can have my own test yaml (e.g. bundles) in repo and like this it will be removed.

Reply:
https://github.com/search?q=org%3Acanonical%20%40git%20clean%20-ffXd%20-e%20%27!.idea%27%20-e%20%27!.vscode%27&type=code

Does not this our practice to clean the files?

Reply:
I see, maybe we should not do it. What do you say?